python/processors/Condor/SubmitLPC.py

```diff
@@ -97,8 +97,9 @@ def ConfigList(config):
             "era" : temp_era,
         }
         if process[stripped_entry[0]]["isData__"]:
+            foundEra = re.search("Period",stripped_entry[0])
```

Comment:
What happened when there are no "Period" in the name? Maybe need to add a protection.

Reply:
@ahenckel Can you follow up this ?